docs/model_pipeline.rst

```diff
@@ -93,9 +105,35 @@ Time series
    :file: ../oemof_b3/schema/timeseries.csv
 
 
-A few more conventions are important to know. Missing data is left empty. If a value applies to all
-regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
-There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+.. _conventions_resources_label_:
+
+Conventions
+^^^^^^^^^^^
+
+A few more conventions are important to know:
+
+* Missing data is left empty.
+* There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+
+* The parameters :attr:`id_scal` and :attr:`id_ts` are optional and will be added automatically if you do not specify them.
+
+* The parameter :attr:`scenario_key` allows you to pass values that are associated with a specific scenario.
+
+* The parameter :attr:`name` must be specified in a certain fixed concatenation of parameters: :attr:`region`-:attr:`carrier`-:attr:`tech` (example: :attr:`B-biomass-g`).
+
+* If a value applies to all regions, :attr:`region` corresponds to :attr:`ALL` for this value.
+* If :attr:`region` is set to :attr:`ALL` in the model-own assumptions, :attr:`name` is to be left blank. The name will be automatically added per region modelling the energy system.
+* If a value applies to the sum of regions, :attr:`region` corresponds to :attr:`TOTAL` for this value.
```

Comment:
This too can be described in the schema (Shorter: ALL if applies to all regions, TOTAL if applies to sum of regions)

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/84b62fce7afc0ddd6b756d8a2d903990badc0f7c.